app/views/metadata_fields/_medical_safety_alerts.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= f.label :alert_type %>
+<%= f.select :alert_type, f.object.facet_options(:alert_type), {include_blank: true}, { class: 'form-control' } %>
```

Comment:
Our [styleguide says](https://github.com/alphagov/styleguides/blob/master/ruby.md#general) we should put spaces inside `{` and `}`.